triage/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM jamiehewland/alpine-pypy:2
+
+RUN apk add --no-cache jq
+RUN curl -o installer https://sdk.cloud.google.com && bash installer --disable-prompts --install-dir=/ && rm installer && ln -s /google-cloud-sdk/bin/* /bin/
+
+ADD *.css *.py *.html *.js update_summaries.sh /
```

Comment:
Any reason to do `/` instead of setting a `WORKDIR`? I might make a folder called `triage/bin` or something and then the image breaks I think.